packages/rum-core/src/domain/configuration.ts

```diff
@@ -73,14 +76,17 @@ export function validateAndBuildRumConfiguration(
     return
   }
 
+  const trackFrustrations = isExperimentalFeatureEnabled('frustration-signals') && !!initConfiguration.trackFrustrations
+
   return assign(
     {
       applicationId: initConfiguration.applicationId,
       version: initConfiguration.version,
       actionNameAttribute: initConfiguration.actionNameAttribute,
       replaySampleRate: initConfiguration.replaySampleRate ?? 100,
       allowedTracingOrigins: initConfiguration.allowedTracingOrigins ?? [],
-      trackInteractions: !!initConfiguration.trackInteractions,
+      trackInteractions: !!initConfiguration.trackInteractions || trackFrustrations,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it could be interesting to add a test case for this one

Reply:
There is already one: see `implies "trackInteractions"`

Reply:
Yep, I resolved this comment after you clarified that for me 🙂